latest/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Redirecting</title>
+  <script>
+    var currentVersion = window.location.pathname.split("/")[1];
+    var xhr = new XMLHttpRequest();
+   
+    xhr.open("GET", "../versions.json");
+    xhr.onload = function () {
+      let versions = JSON.parse(this.responseText);
+
+      let res = versions.find(it => {
+        return it["aliases"].find(alias => { return alias == currentVersion; });
+      });
```

Comment:
It would be nice to add error handling here just in case the alias isn't found. You could probably open an `alert` with an error message.